notifications/models.py

```diff
@@ -0,0 +1,46 @@
+from django.db import models
+
+from users.models import Member
+from django.utils import timezone
+from datetime import timedelta
+
+# TODO:
+# Consider merging notifications of the same type. How do we do this???
+# COULD MERGE BY URL
+
+# Create your models here.
+class Notification(models.Model):
+	member = models.ForeignKey(Member, related_name='notifications_owned', on_delete=models.CASCADE)
+	notify_type = models.CharField(max_length=32)
+	url = models.CharField(max_length=512)
+	content = models.TextField(max_length=8192)
```

Comment:
damn daniel, back at it again with the powers of 2 :D

Reply:
Not sure we need 8192 characters here btw :o

Reply:
Because it can contain a Message of length 4096 along with the starting text ("you got a message from"), and it's got to be a power of two... :field_hockey: